tools/myst/host/syscall.c

```diff
@@ -128,9 +128,39 @@ long myst_fcntl_setlkw_ocall(int fd, const struct flock* arg)
     RETURN(fcntl(fd, F_SETLK, arg));
 }
 
+static __inline__ void sleep_msec(uint64_t milliseconds)
```

Comment:
This is my fault for not putting ``myst_sleep_msec()`` into the ``libmystutils.a``, but there are multiple copies of this function.